cob_generic_can/CMakeLists.txt

```diff
@@ -1,27 +1,30 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cob_generic_can)
 
-find_package(catkin REQUIRED COMPONENTS cob_utilities libntcan libpcan)
+find_package(catkin REQUIRED COMPONENTS socketcan_interface cob_utilities libntcan libpcan)
 
 catkin_package(
-  CATKIN_DEPENDS cob_utilities libntcan libpcan
+  CATKIN_DEPENDS socketcan_interface cob_utilities libntcan libpcan
   INCLUDE_DIRS common/include
-  LIBRARIES ${PROJECT_NAME}_peaksysusb ${PROJECT_NAME}_peaksys ${PROJECT_NAME}_esd
+  LIBRARIES ${PROJECT_NAME}_peaksysusb ${PROJECT_NAME}_peaksys ${PROJECT_NAME}_esd ${PROJECT_NAME}_socketcan
 )
 
 ### BUILD ###
 include_directories(common/include ${catkin_INCLUDE_DIRS})
+include_directories(../ros_canopen/socketcan_interface/include ${catkin_INCLUDE_DIRS})
```

Comment:
NoGo: relative path!
Is this needed, if you have `socketcan_interface` as a dependency and `${catkin_INCLUDE_DIRS}` added?